README.md

```diff
@@ -77,23 +77,24 @@ A non-exhaustive collection of third-party clients and mods for Discord.
 
 | Name | Features | Language(s) | Development Status |
 | :---: | :---: | :---: | :---: |
-| Acord Premium | A client you have to pay for that stole its code. | [Closed source] | ⛔ sus behavior (Stolen code) |
+
```

Comment:
Remove this newline as it breaks the table 
![image](https://github.com/Discord-Client-Encyclopedia-Management/Discord3rdparties/assets/40632486/f4762791-1342-4143-8be8-f0e10e2dbf65)